src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const fetchPeople = async () => {
+    setIsLoading(true);
+
+    try {
+      const peopleResponse: Person[] = await getPeople();
```

Comment:
`getPeople` already have this type
```suggestion
      const peopleResponse = await getPeople();
```